src/themes/five/src/components/Cart/index.js

```diff
@@ -394,6 +396,16 @@ const CartUI = (props) => {
                     </tr>
                   </tbody>
                 </table>
+                {[3, 4].includes(orderState?.options?.type) && (
```

Comment:
please don't use this validation because is hardcoded, create a constant for this